processing/data_collection/gazette/spiders/sp_jau.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+from scrapy import Spider
+from gazette.items import Gazette
+from datetime import datetime
+import dateparser
+
+
+class SpJauSpider(Spider):
+    TERRITORY_ID = "3525300"
+    name = "sp_jau"
+    allowed_domains = ["jau.sp.gov.br"]
+    start_urls = ["http://www.jau.sp.gov.br/jornal-oficial"]
+
+    def parse(self, response):
+        gazettes_selector = response.xpath("//div[@id='concursos']")
+
+        for gazette_selector in gazettes_selector:
+            date = dateparser.parse(
+                gazette_selector.xpath(".//p/text()")
+                .get()
+                .strip()
+                .split(" ")[0]
+                .strip(),
+                date_formats=["%d/%m/%Y"],
+            ).date()
```

Comment:
What do you think about splitting it into different steps? It could improve the readability on being more clear about what function call is doing and by that helping future maintainers. Like:

```python
date_element = gazette_selector.xpath(".//p/text()").get()
date_values = date_element.strip().split(" ")

[...]
```

Reply:
Nice suggestion, done!